dbt_copilot_helper/utils/aws.py

```diff
@@ -163,3 +163,12 @@ def check_response(response):
             fg="red",
         )
         exit()
+
+
+def get_codestar_connection_arn(app_name):
+    client = boto3.client("codestar-connections")
+    response = client.list_connections()
```

Comment:
Could inline this...
```suggestion
    response = boto3.client("codestar-connections").list_connections()
```